core/cont/src/TClassTable.cxx

```diff
@@ -590,15 +649,22 @@ DictFuncPtr_t TClassTable::GetDict(const char *cname)
 
 DictFuncPtr_t TClassTable::GetDict(const std::type_info& info)
 {
-   if (!CheckClassTableInit()) return nullptr;
+   if (!CheckClassTableInit())
+      return nullptr;
+
+   if (gDebug > 9)
+      gROOT; // Info migth recursively call TClassTable during the gROOT init
```

Comment:
I think that using `ROOT::GetROOT()` is much more readable than just spelling the `gROOT` PP macro (same for all uses below).

Reply:
I agree it is.